Misc/NEWS.d/next/Core and Builtins/2022-05-23-17-38-04.gh-issue-93071.NHsmc8.rst

```diff
@@ -0,0 +1 @@
+Removed an unnecessary assignment in the initialization of a loop in ``long_lshift1()`` in ``longobject.c``.
```

Comment:
Honestly, I don't see the benefit to communicate this to Python end users. If it would make ``a << b`` way faster, it would be worth to mention it. But I don't see how I could make it faster. I suggest to remove the NEWS entry.

Reply:
@vstinner Sure, I can remove it. Thanks!